projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -752,16 +765,130 @@ function wpcom_add_new_page_check( $post_id, $post ) {
+ *
+ * @return int|null The post ID of the 'About' page if it exists, null otherwise.
+ */
+function wpcom_get_site_about_page_id() {
+	$annotation = wpcom_get_theme_annotation( get_stylesheet() );
+
+	// Return null if there is no annotation or the annotation doesn't have any content.
+	if ( ! $annotation || ! isset( $annotation['content'] ) || ! is_array( $annotation['content'] ) ) {
+		return null;
+	}
+
+	// Use the annotation to build up a list of 'About' pages.
+	$headstart_about_pages = array_filter(
+		$annotation['content'],
+		function ( $post ) {
+			if ( 'page' !== $post['post_type'] ) {
+				return false;
+			}
+
+			if ( 'about' === $post['post_name'] || false !== strpos( $post['post_title'], 'About' ) ) {
+				return true;
+			}
+		}
+	);
+
+	// Return null if there are no 'About' pages.
+	if ( empty( $headstart_about_pages ) ) {
+		return null;
+	}
+
+	// Get the hs_old_ids for the 'About' pages. We'll use these to find any published pages with the same hs_old_id.
+	$headstart_about_page_hs_old_ids = array_map(
+		function ( $about_page ) {
+			return $about_page['hs_old_id'];
+		},
+		$headstart_about_pages
+	);
+
+	// Return null if there aren't any hs_old_ids to check.
+	if ( empty( $headstart_about_page_hs_old_ids ) ) {
+		return null;
+	}
+
+	$filters     = array(
+		'post_type'    => 'page',
+		'meta_key'     => '_hs_old_id',
+		'meta_compare' => 'IN',
+		'meta_value'   => $headstart_about_page_hs_old_ids,
+	);
+	$about_pages = get_posts( $filters );
+
+	// Return null if we couldn't find any published pages matching the hs_old_ids.
+	if ( empty( $about_pages ) ) {
+		return null;
+	}
+
+	// Return the id of the first About page.
+	return $about_pages[0]->ID;
+}
+
+/**
+ * Determine `update_about_page` task visibility. The task is visible if there is an 'About' page on the site.
+ *
+ * @return bool True if we should show the task, false otherwise.
+ */
+function wpcom_is_update_about_page_task_visible() {
+	return wpcom_get_site_about_page_id() !== null;
+}
+
+/**
+ * When a page is updated, check to see if it's the About page and mark the task complete accordingly.
+ *
+ * @param int     $post_id The ID of the post being updated.
+ * @param WP_Post $post The post object.
+ */
+function wpcom_update_about_page_check( $post_id, $post ) {
+	// Don't do anything if the task is already complete.
+	if ( wpcom_is_task_option_completed( array( 'id' => 'update_about_page' ) ) ) {
+		return;
+	}
+
+	// We only care about pages, ignore other post types.
+	if ( $post->post_type !== 'page' ) {
+		return;
+	}
+
+	// Ensure that Headstart posts don't mark this as complete
+	if ( defined( 'HEADSTART' ) && HEADSTART ) {
+		return;
+	}
+
+	// We only care about published pages. Pages added via the API are not published by default.
+	if ( $post->post_status !== 'publish' ) {
+		return;
+	}
+
+	// If the page is not the about page, ignore it.
+	if ( $post->post_title !== 'About' ) {
```

Comment:
Does this work if we're in a different locale? It may be worth noting that as a comment if it's true, as this looks like it will break for any translated content.

Reply:
Good catch, this is actually a bug. What we really need to do here is make sure that the page being updated has an option matching one of the `hs_old_id`s that denotes if the page is an 'About' page.

Once I've added the caching you suggested about, we should be able to leverage that here to check the page being updated. It's probably worth adding some caching here as well to ensure we're not pulling the same page options over and over.